### PR TITLE
Create react app

### PR DESCRIPTION
**Pull request by elyor-doniyorov**
### React Tutorial

**In this PR:**

- Use JavaScript events.
- Use React life cycle methods.
- Understand the mechanism of lifting state up.

### CODE REVIEW

**Review by elmar8287 — APPROVED** [review]
Hi @elyor-doniyorov , 👋

You have done an amazing job so far! 👏👏👏😉 😉 !

To Highlight
✔️ Youimplement all components💪
✔️ Style as in the template
✔️ Add pages 😉

STATUS: Approved 🥇 

Optional: The README file will look better if you delete the boilerplate readme that comes with npx create-react-app app-name

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: src/classBased/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Header = () => {
+  const headerStyle = {
+    padding: '20px 0',
+    lineHeight: '1.5em',
+  };
+
+  return (
+    <header style={headerStyle}>
```
Comment (elmar8287) [inline src/classBased/components/Header.js:10]: I liked how you use const for creating the style for future using

File: src/classBased/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        style={{
+          fontSize: '6rem',
+          fontWeight: '600',
+          marginBottom: '2rem',
+          lineHeight: '1em',
+          color: '#ececec',
+          textTransform: 'lowercase',
+          textAlign: 'center',
+        }}
+      >
```
Comment (elmar8287) [inline src/classBased/components/Header.js:21]: Also inline styling, as it requered for React basic styling

**Review by elmar8287 — COMMENTED** [review]
👍 for using React `basic` styling

File: src/classBased/components/Header.js (hunk 2 above)
Comment (elyor-doniyorov) [reply src/classBased/components/Header.js:21]: Hello Elmar, thank you for your review!

**Pull request merged**